timemachine/cpp/src/nonbonded.cu

```diff
@@ -18,6 +18,9 @@
 #include <fstream>
 #include <streambuf>
 
+
+#define SORT_INTERVAL 100
```

Comment:
lets call this `SPATIAL_SORT` or `HILBERT_SORT` for more clarity? There's a large number of different ways in which we sort.